### PR TITLE
[CHK-2815] Add mock GET psps by wallet id

### PR DESCRIPTION
**Pull request by petretiandrea**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -24,3 +24,6 @@ export const getSessionIdCookie: (req: express.Request) => string = req =>
     maybeGetSessionIdCookie(req),
     O.getOrElse(() => "")
   );
+
+export const waitTime: (timeMillis: number) => Promise<void> = time =>
+  new Promise(resolve => setTimeout(() => resolve(), time));
```
Comment (giovanniberti) [inline src/utils.ts]: This is defined but not referenced anywhere, should we leave it?

File: src/handler/get-psps.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  logger.info(
+    `[Retrieve psps from walletId ${walletId}] - Return success case`
+  );
+
+  return res.send({
+    belowThreshold: false,
+    bundleOptions: [
+      {
+        abi: "32532",
+        bundleDescription: "Bundel description",
```
Comment (giovanniberti) [inline src/handler/get-psps.ts]: typo
```suggestion
        bundleDescription: "Bundle description 1",
```

File: src/handler/get-psps.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        bundleName: "Bundle name",
```
Comment (giovanniberti) [inline src/handler/get-psps.ts]: Number
```suggestion
        bundleName: "Bundle name 1",
```

File: src/handler/get-psps.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        idBrokerPsp: "123456",
+        idBundle: "123",
+        idChannel: "123",
+        idCiBundle: "123",
+        idPsp: "123",
+        onUs: false,
+        paymentMethod: "paypal",
+        primaryCiIncurredFee: 0,
+        pspBusinessName: "Business Name",
```
Comment (giovanniberti) [inline src/handler/get-psps.ts]: Number
```suggestion
        pspBusinessName: "Business Name 1",
```

**Review by giovanniberti — APPROVED** [review]

File: src/utils.ts (hunk 1 above)
Comment (petretiandrea) [reply src/utils.ts]: My bad

**Pull request merged**